src/patternfly/components/Select/select.scss

```diff
@@ -127,6 +123,11 @@
   width: 100%;
 }
 
+.pf-c-select__toggle:not(.pf-m-plain),
+.pf-c-select__menu {
+  @include pf-t-light; // other than a plain select toggle, the select component should be a light theme
```

Comment:
Was there a way to disable the dark-mode icon in pf so that if its only meant to be light, the consumer can't see it like this:

<img width="514" alt="Screen Shot 2019-07-15 at 4 37 45 PM" src="https://user-images.githubusercontent.com/20118816/61247438-10408780-a71f-11e9-8f80-460f3434b112.png">

Reply:
Ah good catch, I forgot to test that. I changed the theme since I assumed the plain variation would have a dark background and white text in a dark theme, but assuming the select will be used in a form with other form controls, I suppose this variation should always follow the light theme, too. Updated the PR.

Reply:
yep good idea until we decide that it should be used in dark mode